releasenotes/notes/serialize-primitive-job-aa97d0bf8221ea99.yaml

```
@@ -0,0 +1,5 @@
+---
+features_primitives:
+  - |
+    To make :class:`.PrimitiveJob` serializable, ``qiskit.primitives.containers.DataBin`` has been 
```

I suggest the following because `DataBin` is public now.
https://docs.quantum.ibm.com/api/qiskit/qiskit.primitives.DataBin
```suggestion
    To make :class:`.PrimitiveJob` serializable, :class:`.DataBin` has been 
```